source/manual/alerts/rabbitmq-high-number-of-unprocessed-messages.html.md

```diff
@@ -0,0 +1,45 @@
+---
+owner_slack: "#govuk-2ndline"
+title: 'RabbitMQ: High number of unprocessed messages'
+parent: "/manual.html"
+layout: manual_layout
+section: Icinga alerts
+last_reviewed_on: 2020-09-07
+review_in: 6 months
+---
+
+For information about how we use RabbitMQ, see [here][rabbitmq_doc].
+
+We check that there is not a significant build up of messages on certain queues
+as we expect messages to be processed quickly. The queues this alert applies to
+are: `email_alert_service`, `email_unpublishing` and `cache_clearing_service-*`
```

Comment:
Could we add a link to verify this?

Reply:
Sure, I've added links to the config for each queue.